code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -266,7 +266,7 @@
 	fire_delay = 1
 
 /obj/item/weapon/gun/projectile/automatic/ar
-	name = "ARG"
+	name = "\improper NT-ARG 'Boarder'"

```

Comment:
I did want to put it like that, but then I was informed what ARG actually means. So I just feel it would sound a bit....redundant.